Tasks/KubernetesManifestV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -11,7 +11,10 @@
   "loc.input.help.namespace": "Sets the namespace for the commands by using the –namespace flag. If the namespace is not provided, the commands will run in the default namespace.",
   "loc.input.label.strategy": "Strategy",
   "loc.input.help.strategy": "Deployment strategy to be used",
+  "loc.input.label.trafficSplitMethod": "Traffic split method",
+  "loc.input.help.trafficSplitMethod": "Deployment strategy to be used",
```

Comment:
Label and help are looking like related. Can we add some more details in 'Help'.

Reply:
i will check with PM and will add.